lib/MigrationTask.js

```diff
@@ -0,0 +1,73 @@
+const Task = require('./Task');
+
+module.exports = class MigrationTask extends Task {
+
+    constructor(task, rerun = false) {
```

Comment:
can u add some comments to the functions and there documenting the parameter eg the taks parameter needs to be an instance of tasks